Gopkg.toml

```diff
@@ -26,13 +26,9 @@
 
 
 [[constraint]]
-  branch = "master"
+  branch = "teemow-remove-aws-elb-settings"
```

Comment:
https://github.com/giantswarm/apiextensions/pull/151

this removes unused apiextension types